bokeh/models/graphs.py

```diff
@@ -161,6 +162,19 @@ class EdgesAndLinkedNodes(GraphHitTestPolicy):
 
     pass
 
+class NodesAndAdjacentNodes(GraphHitTestPolicy):
+    '''
+    With the ``NodesAndAdjacentNodes`` policy, inspection or selection of
+    graph nodes will result in the inspection or selection of the node and
+    those directly connected to it by edges. There is no direct selection
+    or inspection of graph edges or nodes. The last entry in each list
```

Comment:
This wording is a little confusing to me. Maybe something like:
```suggestion
    With the ``NodesAndAdjacentNodes`` policy, inspection or selection of
    graph nodes will also result in the inspection or selection any nodes that
    are immediately adjacent (connected by a single edge). There is no selection
    or inspection of graph edges. The last entry in each list
```

cc @tcmetzger for suggestions also

Reply:
Your version does sound more readable, I will change the description accordingly. :)